.github/workflows/run-unit-tests.yml

```diff
@@ -34,11 +34,21 @@ jobs:
           brew install colima docker
           colima start
 
+          # For testcontainers to find the Colima socket
+          # https://github.com/abiosoft/colima/blob/main/docs/FAQ.md#cannot-connect-to-the-docker-daemon-at-unixvarrundockersock-is-the-docker-daemon-running
+          sudo ln -sf $HOME/.colima/default/docker.sock /var/run/docker.sock
+
       - name: Build Environment
         run: make build
 
-      - name: Run Tests
-        run: poetry run pytest --cov=agenthub --cov=opendevin --cov-report=xml ./tests/unit
+      # Sandbox requires to import modules in fixture to change os.environ
+      # imports in other test files will cause issues, so we need to test sandbox separately
```

Comment:
If it's a problem of `import`, you could move those imports to the fixture definition method?

Reply:
Do you mean to move the import for other unit tests into a fixture? I think we can do that, but it will probably look a bit messy. I can definitely give it a try though.

Reply:
Oh maybe I misunderstood this comment. Do you mind elaborating the issue?

Reply:
Currently (i think) OpenDevin first reads in a config (from env var) when the first import happen, and will use this config to construct the Sandbox for testing.

`test_sandbox.py` initializes different sandboxes by setting different environment variable before importing sandbox from OpenDevin, making sure the sandbox is initialized with proper config.

However, if we run `test_sandbox.py` together with other tests, other tests would import OpenDevin first (causing `config` to be already initialized), and making the environment variable setting in `test_sandbox.py` no longer working.

I'm not super familiar with pytest so i'd love to hear if you have any suggestion to get this work properly without doing a separate pytest command :)

Reply:
Aha I see, so the curse comes from global variables :( I don't have a solution to that.

Reply:
wait but could you simply overwriting the global variables in your sandbox test?

Reply:
I think patching `config.config` might do the work but I'm not quite sure how that'd work! let me give it a try though!

Reply:
well no really, patching changing `config.config` before running the code would not work (probably bc. some synchronous issue)

Reply:
ahh i figure out a way to do it by patching it within test cases, let's see if it works